parlai/tasks/msc/mutators.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import random
+from typing import List
+from parlai.core.message import Message
+from parlai.core.mutators import register_mutator, ManyEpisodeMutator
+
+
+@register_mutator("ltm_mutator")
+class LongTermMemoryMutator(ManyEpisodeMutator):
+    """
+    Replaces the episode labels in messages with "personal_knwowledge".
+
+    Episodes are flattened to ensure dialogue history is maintained appropriately.
+    """
+
+    def many_episode_mutation(self, episode: List[Message]) -> List[List[Message]]:
+        history = []
+        for message in episode:
+            if message['text'] == '__SILENCE__':
```

Comment:
one nit: replace this `'__SILENCE__'` with `DUMMY_TEXT`

Reply:
happy to do this, i think i hesitated at first because it would require a nested import (to avoid circular dependencies), but should be fine for me!

Reply:
ah didn't realize it would require a nested import. I'm fine either way.

Reply:
ok, if you're ok with it i'll avoid the nested import for now, then